src/Tribe/Service_Providers/Context.php

```diff
@@ -284,6 +302,11 @@ static function ( $post_type ) {
 				],
 				'venue_post_type'      => [
 					'read' => [
+						Tribe__Context::FUNC          => [
+							static function () {
+								return ! empty( tribe_get_request_var( Venue::POSTTYPE, false ) ) ?: Tribe__Context::NOT_FOUND;
```

Comment:
:no_entry_sign: Using short ternaries is not allowed

<sub>hash: 252f2a661e7328091d253fc702c8911c</sub>